en-US/ranking/criteria.md

```diff
@@ -2,78 +2,173 @@
+* Rules are, literally, rules that may not be broken under any circumstances.
+* Guidelines try to guide the mapper and may be ignored when deemed reasonable.
+
+If you're looking for the steps of getting a mapset ranked, see: [Quaver Ranking Process](/docs/Ranking/Process).
+
+# Metadata
+
+#### Rules
+
+* **Metadata must be consistent across difficulties.** The only exception being specific map-descriptive tags, such as pattern names, that improve the searchability of singular difficulties.
+
+## Artist
+
+#### Rules
+
+* **The artist must be absolutely correct.**
+* **The artist must be romanized.** Non-romanized artist can be added to tags.
+* **The artist must not repeat in tags.** Only exception being the case where the artist appears in a relevant tag, such as song genre.
+* If there is no existing person to credit as the artist, `Unknown Artist` must be used.
+
+## Title
+
+#### Rules
+
+* **The title must be absolutely correct.**
+* **The title must be romanized.** Non-romanized title can be added to tags. Loan words from other languages should use the original words instead of attempted romanization.
+* **The title must not repeat in tags.** Only exception being the case where the song title appears in a relevant tag, such as song genre.
+
+## Source
+
+#### Rules
+
+* **The source field must be used if the song is directly from a video game, album, series etc.**
+* **Remixes, arrangements, covers etc. that are based on another song should use the original song's source.**
+* **Song compilations, medleys and other pieces of media with multiple songs without a common source should have the sources put in the tags.** This rule does not apply if the piece of media is linked to a source itself, such as an album.
+* **The source must be romanized** Non-romanized source can be added to tags.
+
+## Tags
+
+#### Rules
+
+* **The tags must contain the genre of the song.**
+* **The tags must be comma-separated.** This is to ensure that tags consisting of multiple words can be easily distinguished.
+
+#### Guidelines
+
+* Including relevant tags such as language, instruments or metadata translations is encouraged.
+
+## Standardization
+
+The following metadata rules are put in place to create consistency within the ranked section.
+
+* Any form of `vs.`, `VS`, etc. indicating collaboration between artists should be formatted as `vs.`.
+* Any form of `feat.`, `ft.`, etc. indicating a featured artist should be formatted as `feat.`.
+* If a fictional character is credited as the artist of a song, the artist should be formatted as `Character (CV: Voice Actor)`
+* If a song is used in a TV program, series, or similar media as an opening/ending song, `(TV Size)` marker must be used at the end of the title. If the title contains similar marker by default, it should be replaced with `(TV Size)`.
+* If a song title contains short version or game version markers by default, they must be replaced with `(Short Ver.)` and `(Game Ver.)` respectively.
```

Comment:
Similar to my other comment, this kind of standardisation should only be applied in the event that the original artist did not indicate anything, and we deem that an indication is necessary (ex: anime openings that officially don't have any markers to indicate that they are openings, despite having both an opening version and full version of the song).

We shouldn't be editing the markers that the artists decided to put there - let them decide what they want their own songs to be called.